test/test-utils/helpers/overridesHelper.ts

```diff
@@ -188,12 +189,29 @@ export class OverridesHelper extends GameManagerHelper {
   }
 
   /**
-   * Override each wave to not have random trainer battles
+   * Override the chance of encountering a random enemy trainer
+   * @param trainerChance - `0` to disable enemy trainer spawns, `1` to guarantee an enemy trainer spawn
+   * @see {@linkcode Overrides.RANDOM_TRAINER_CHANCE_OVERRIDE} for a more complete description of this override
+   * @returns `this`
+   */
+  public trainerChance(trainerChance: number): this {
+    vi.spyOn(Overrides, "RANDOM_TRAINER_CHANCE_OVERRIDE", "get").mockReturnValue(trainerChance);
+    if (trainerChance === 0 || trainerChance === 1) {
+      this.log(`Trainer chance override set to ${trainerChance}!`);
+    } else {
+      this.log(`Trainer chance override set to 1 / ${trainerChance}!`);
+    }
+    return this;
+  }
+
+  /**
+   * Override each random enemy trainer to be of a given type
+   * @param trainerType - The {@linkcode TrainerType} to set
    * @returns `this`
    */
-  public disableTrainerWaves(): this {
-    vi.spyOn(Overrides, "DISABLE_RANDOM_TRAINERS_OVERRIDE", "get").mockReturnValue(true);
-    this.log("Random trainer waves are disabled!");
+  public trainerType(trainerType: TrainerType): this {
+    vi.spyOn(Overrides, "TRAINER_TYPE_OVERRIDE", "get").mockReturnValue(trainerType);
+    this.log(`Trainer type override set to ${TrainerType[trainerType]} (=${trainerType})!`);
```

Comment:
NIT: we never use the override word in other logs

```suggestion
    this.log(`Trainer type set to ${TrainerType[trainerType]} (=${trainerType})!`);
```